test/lib/alternator_test_env.hh

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2021 ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <seastar/core/sharded.hh>
+#include <seastar/core/future.hh>
+
+#include "alternator/executor.hh"
+#include "utils/rjson.hh"
+#include "cql3/query_processor.hh"
+#include "service/storage_service.hh"
+#include "service/storage_proxy.hh"
+#include "cdc/log.hh"
+#include "cdc/metadata.hh"
+#include "cdc/generation.hh"
+#include "db/system_distributed_keyspace.hh"
+
```

Comment:
I'm sure most of these can be replaced with forward declarations. Ref #1.

Reply:
lemme try